recipes/openssh/openssh.inc

```diff
@@ -72,13 +72,20 @@ inherit auto-package-utils
 AUTO_PACKAGE_UTILS = "ssh-add ssh-agent ssh-keyscan ssh-keygen ssh-keysign \
 	scp ssh sshd sftp sftp-server ssh-pkcs11-helper"
 
+PACKAGES_${PN} += "ssh-config sshd-config"
```

Comment:
Although not really a requirement, normally we prefix all packages with ${PN}, and I don't see a special reason to do it different here.